src/fontra/views/editor/panel-designspace-navigation.js

```diff
@@ -121,7 +121,7 @@ export default class DesignspaceNavigationPanel extends Panel {
     );
   }
 
-  attach() {
+  connectedCallback() {
```

Comment:
This should be done in the constructor. Please only use connectedCallback() for things that _must_ be done at connect time.